plugins/mcreator-core/blockly/mcreator_blocks.js

```diff
@@ -149,6 +149,34 @@ Blockly.Blocks['text_contains'] = {
     }
 };
 
+Blockly.Blocks['text_starts_with'] = {
+    init: function () {
+        this.appendValueInput('text').setCheck('String')
+            .appendField(javabridge.t("blockly.block.text_contains.in"));

```

Comment:
You should either rename this localization key, or use separate keys for the "Starts/ends with" blocks